tests/pamiq_core/console/test_cui.py

```diff
@@ -0,0 +1,212 @@
+import json
+import re
+import sys
+from collections.abc import Generator
+from unittest.mock import MagicMock, patch
+
+import httpx
+import pytest
+
+from pamiq_core.console.cui import Console, main
+
+
+class TestConsole:
+    @pytest.fixture
+    def mock_httpx(self) -> Generator[MagicMock, None, None]:
+        # Mock httpx module imported in pamiq_core/console/cui.py
+        with patch("pamiq_core.console.cui.httpx") as mock_httpx:
+            # Mock GET response
+            mock_response = MagicMock()
+            mock_response.text = json.dumps({"status": "running"})
+            mock_httpx.get.return_value = mock_response
+            # Mock POST response
+            api_response = MagicMock()
+            api_response.text = json.dumps({"result": "success"})
+            mock_httpx.post.return_value = api_response
+            yield mock_httpx
+
+    @pytest.fixture
+    def console(self, mock_httpx: MagicMock) -> Console:
+        return Console(host="localhost", port=8391)
+
+    def test_initial_prompt(self, mock_httpx: MagicMock) -> None:
+        mock_httpx.get.return_value.text = json.dumps({"status": "running"})
+        console = Console("localhost", 8391)
+        assert console.prompt == "PAMIQ-console (running) > "
+
+    def test_initial_prompt_when_offline(self, mock_httpx: MagicMock) -> None:
+        mock_httpx.RequestError = httpx.RequestError
+        mock_httpx.get.side_effect = httpx.RequestError("Test RequestError")
+        console = Console("localhost", 8391)
+        assert console.prompt == "PAMIQ-console (offline) > "
+
+    def test_get_all_commands(self, console: Console) -> None:
+        assert set(console.get_all_commands()) == {
+            attr_name[3:] for attr_name in dir(console) if attr_name.startswith("do_")
+        }
```

Comment:
これは結局実装と同様のロジックを用いて等価かどうかを検証しているから、テストになっていないんだよな...
本当にちゃんとテストするなら、右辺は完全に明示すると良い。
```suggestion
        assert set(console.get_all_commands()) == {
             "h", "help", ...
        }
```

Reply:
https://github.com/MLShukai/pamiq-core/pull/210/commits/82ae2f38e6298bc952e4225eebf98cf3e5cab46f 
完全に明示するようにした